importer.py

```diff
@@ -1255,10 +1320,17 @@ def begin_child(self, context, type, name, values):
 
         #if type in ('SCALAR', 'CHAR*', 'BILLBOARD', 'DCS', 'DART', 'SWITCH', 'OBJECTTYPE', 'TAG', 'MODEL', 'TEXLIST', 'REF'):
 
+        if type in ('SWITCHCONDITION', 'SWITCH'):
+            return EggSwitchNode()
+
+        if type in ('DCS'):
+            # DCS tags won't have a name
+            self.properties[type] = values[0]
```

Comment:
This checks whether the type is one of D, C, or S, so this doesn't work as intended.